python/tests/examples.py

```diff
@@ -280,38 +277,41 @@ def test_relation_based_on_preexisting_hypothesis(self):
         self.dump_graph(hypo_g, "Example of relation based on pre-existing hypothesis (hypothesis)")
 
         # print normal graph
-        self.new_file(g, "test_relation_based_on_preexisting_hypothesis.ttl")
         self.dump_graph(g, "Example of relation based on pre-existing hypothesis")
 
+        # when validating, need both graphs
+        for s, p, o in hypo_g:
```

Comment:
It looks like you are repeating this code fragment. We should pull it out into a separate method.

Reply:
Apparently, this can be done with the `+=` operator. Fixed

Reply:
That works too.